docs/content/tardigrade.md

```diff
@@ -1,4 +1,4 @@
----
+s---
```

Comment:
This looks like a mistake?

Reply:
Darn it. Yes, its probably me attempting to do a quick Ctrl+s - without pressing the Ctrl properly. Happens now and then.. Good that you spotted it!